### DISCUSSION TITLE
All sectors faulty

### DISCUSSION THREAD

**Discussion by ktg49292** [filecoin-project/lotus#6242] [Storage Provider] [answered]
**I have lost all power. Somebody help me!**

miner.log:11759:2021-05-13T09:34:00.831+0900	WARN	storageminer	storage/wdpost_run.go:226	Checked sectors	{"checked": 20, "good": 20}
miner.log:11760:2021-05-13T09:34:01.602+0900	WARN	storageminer	storage/wdpost_run.go:314	declare faults recovered Message CID	{"cid": "bafy2bzaceamzmt25gsm37heibxjcz5noel4vgywwqm54xiqzvriaclmze5d2w"}
miner.log:11951:2021-05-13T10:34:00.299+0900	WARN	storageminer	storage/wdpost_run.go:226	Checked sectors	{"checked": 20, "good": 20}
miner.log:11952:2021-05-13T10:34:00.325+0900	INFO	storageminer	storage/wdpost_run.go:584	running window post	{"chain-random": "hI7K1zbxYElRjT0KSel/BEIVhuKP6Foufm1k1QEFcls=", "deadline": {"CurrentEpoch":752108,"PeriodStart":752118,"Index":0,"Open":752118,"Close":752178,"Challenge":752098,"FaultCutoff":752048,"WPoStPeriodDeadlines":48,"WPoStProvingPeriod":2880,"WPoStChallengeWindow":60,"WPoStChallengeLookback":20,"FaultDeclarationCutoff":70}, "height": "752108", "skipped": 0}
miner.log:12141:2021-05-13T11:16:23.946+0900	INFO	storageminer	storage/wdpost_run.go:600	computing window post	{"batch": 0, "elapsed": 2543.620275436}

**every day deadline window posted. but all sector faults!**

**How do I solve the problem?**

**please help me!  T_T**

<img width="1245" alt="스크린샷 2021-05-13 오후 12 04 08" src="https://user-images.githubusercontent.com/82360290/118071759-58946380-b3e3-11eb-89bb-0046176613b1.png">

https://filfox.info/en/message/bafy2bzaced4mvxaxt2gligcumxmro6lm3evxjmlctjwzqn2slsklzhe3r7yk6

**Comment by s0nik42** [comment, marked as answer]
Your miner take to much time to compute the WindowPostT
You should better ask your question on slack #fil-help you will have more people to help

**Reply by ktg49292** [reply]
thank you! replay to me.

I have a Slack account, but I am not subscribed to the filecoin channel. 
How can I get invited? 
slack accout id: ktg4929@hotmail.com

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
![2021-05-13_16-37](https://user-images.githubusercontent.com/60559388/118100976-ac19a800-b409-11eb-89e0-eaa23989eff3.png)

**Reply by gogogogo123** [reply]
hey bro, I want to be invited too
slack accout id: hhui2824@gmail.com
